identity_account_storage/src/storage/traits.rs

```diff
@@ -70,7 +71,7 @@ mod storage_sub_trait {
 #[cfg_attr(not(feature = "send-sync-storage"), async_trait(?Send))]
 #[cfg_attr(feature = "send-sync-storage", async_trait)]
 pub trait Storage: storage_sub_trait::StorageSendSyncMaybe + Debug {
-  /// Creates a new identity for the given `network`.
+  /// Creates a new identity of type `DIDType` for the given `network`.
```

Comment:
```suggestion 
/// Creates a new identity of the type declared in `did_type` for the given `network`.
```

The final line of the method's documentation which currently is: 
```
/// Returns the generated DID and the location at which the key was stored.
```
should also be changed to specify that the returned did is of type `CoreDID` (regardless of `did_type`), i.e.:

```
/// Returns the generated DID represented as a [`CoreDID`] and the location at which the key was stored.
```